### PR TITLE
fix: improved detail popup position in case of long time schedule(fix #592)

### LINKED ISSUE #592: In the weekly view, a detailed pop-up is placed beyond the time grid in case of a long time schedule
## Describe the bug
In the weekly view, a detailed pop-up is placed beyond the time grid in case of a long time schedule

## To Reproduce
1. In the weekly view (`default: 'week'`)
2. Make a long schedule.(e.g. 04/27 00:30 ~ 23:00)
    The category of schedule is 'time'. (`category: 'time'`)
3. Scroll down the time grid.
4. Click this schedule.
5. Check the placement of the detailed popup.
스크린샷 2020-04-27 18 06 32

## Expected behavior
It should be placed inside the time grid.
스크린샷 2020-04-27 18 04 58

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* fix: improved detail popup position in case of long time schedule(fix #592)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/view/popup/scheduleDetailPopup.js [hunk 1]
```diff
@@ -176,11 +176,15 @@ ScheduleDetailPopup.prototype._getYAndArrowTop = function(
     containerTop,
     containerBottom
 ) {
-    var scheduleVerticalCenter = (scheduleBoundTop + scheduleBoundBottom) / 2;
-    var y = scheduleVerticalCenter - (layerHeight / 2);
     var ARROW_WIDTH_HALF = 8;
+    var scheduleVerticalCenter;
+    var y;
     var arrowTop;
```
Comment (seonim-ryu) [inline src/js/view/popup/scheduleDetailPopup.js:182]: 변수를 초기화하는게 아니라면 한 줄에 선언해도 되지 않나요??
```js
var scheduleVerticalCenter, y, arrowTop;
```

Comment (dongsik-yoo) [reply src/js/view/popup/scheduleDetailPopup.js:182]: https://ui.toast.com/fe-guide/ko_CODING-CONVENSION/#%EC%84%A0%EC%96%B8%EA%B3%BC-%ED%95%A0%EB%8B%B9%EC%9D%98-%EB%B6%84%EB%A6%AC%EB%A5%BC-%ED%97%88%EC%9A%A9%ED%95%98%EB%8A%94-%EA%B2%BD%EC%9A%B0-%EC%84%A0%EC%96%B8%EB%A7%8C-%ED%95%98%EB%8A%94-%EB%B3%80%EC%88%98%EB%8A%94-var%EC%9D%84-%ED%95%9C-%EB%B2%88%EB%A7%8C-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94-%EB%B0%A9%EC%8B%9D%EC%9D%84-%ED%97%88%EC%9A%A9%ED%95%9C%EB%8B%A4-es5

File: src/js/view/popup/scheduleDetailPopup.js [hunk 2]
```diff
@@ -176,11 +176,15 @@ ScheduleDetailPopup.prototype._getYAndArrowTop = function(
...
 
+    scheduleBoundTop = scheduleBoundTop < 0 ? 0 : scheduleBoundTop;
```
Comment (dongsik-yoo) [inline src/js/view/popup/scheduleDetailPopup.js:184]: 👍

File: src/js/view/popup/scheduleDetailPopup.js (hunk 1 above)
Comment (jungeun-cho) [reply src/js/view/popup/scheduleDetailPopup.js:182]: 👍

**Pull request merged**